### PR TITLE
94 2.0: Remove support for non-Sparrow Wallets

### LINKED ISSUE #94: 2.0: Remove support for non-Sparrow Wallets
- 2.0 will only support Sparrow to start.
- Lets keep the same data structures to minimize refactor, and ease of adding back in - just comment out the other wallets

### PR DESCRIPTION
**Pull request by kylemccullen**
Closes #94

### CODE REVIEW

File: Branta/Automation/LoadCheckSums.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+namespace Branta.Automation;
+
+public class LoadCheckSums : BaseAutomation
+{
+    public List<BaseWallet> WalletTypes = new();
+
+    private readonly BrantaClient _brantaClient;
+
+    public LoadCheckSums() : base(null, null, 60 * 60 * 4)
```
Comment (kylemccullen) [inline Branta/Automation/LoadCheckSums.cs:12]: CheckSums are loaded on startup, after that reload every 4 hours

File: Branta/Classes/Analytics.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿using CountlySDK;
+using CountlySDK.Entities;
+
+namespace Branta.Classes;
+
+public static class Analytics
```
Comment (kylemccullen) [inline Branta/Classes/Analytics.cs:6]: Leaving the name generic, as we may eventually move on from Countly.

Comment (keith-gardner) [reply Branta/Classes/Analytics.cs:6]: Nice, did the same with these:

https://github.com/BrantaOps/branta-mac/blob/main/branta/Utils/BrantaAnalytics.swift
https://github.com/BrantaOps/branta-mac/blob/main/branta/Utils/BrantaConstants.swift
https://github.com/BrantaOps/branta-mac/blob/main/branta/Utils/BrantaLogger.swift
https://github.com/BrantaOps/branta-mac/blob/main/branta/Utils/BrantaNotify.swift

File: Branta/Classes/Settings.cs [hunk 3]
```diff
@@ -1,9 +1,49 @@
...
 public class Settings
 {
     public ClipboardGuardianSettings ClipboardGuardian { get; set; } = new();
     public WalletVerificationSettings WalletVerification { get; set; } = new();
+
+    public static Settings Load()
+    {
+        return new Settings
+        {
+            ClipboardGuardian = new ClipboardGuardianSettings
```
Comment (keith-gardner) [inline Branta/Classes/Settings.cs:14]: 💯

**Review by keith-gardner — APPROVED** [review]

**Pull request merged**